tutorials/Threshold_Weighted_Scores.ipynb

```diff
@@ -5,14 +5,23 @@
     "\n",
-    "For example, suppose health agencies will start taking preventative action for treating heat illness whenever the temperature forecast for the following day exceeds 35°C and that this preventative action increases with higher temperature forecasts. If the temperature forecast is 20°C and the observed temperature is 10°C, then health agencies will have made the correct decision to not take preventative action, despite the large magnitude in error. In this case, the MSE would be 100°C<sup>2</sup>. However, if we calculate a threshold weighted MSE (twMSE) that places an equal weight on decision thresholds above 35°C and a zero weight on decision thresholds below 35°C, the twMSE will give a value of 0 in this case. If at least one of the forecast value and the observed value is above 35°C and they differ from each other, then there will be a non-zero error.\n",
+    "Threshold-weighted scores contrast with metrics like mean squared error (MSE) and mean absolute error (MAE), which can be interpreted as evaluating performance when equal weight is placed on all decision thresholds ([Ehm 2016](https://doi.org/10.1111/rssb.12154)).\n",
     "\n",
-    "Threshold weighting differs from the `weights` argument that appears in most metrics in `scores` including the threshold weighted scores. The `weights` argument weights the errors at each point before calculating the mean score. Examples including weighting the results based on latitude to account for grid size or by weighting results based on population density. For more information see the [weighting results tutorial](./Weighting_Results.ipynb).\n",
+    "As another example, suppose health agencies will start taking preventative action for treating heat illness whenever the temperature forecast for the following day exceeds 35°C and that this preventative action is increasingly important with even higher temperature forecasts. If the temperature forecast is 20°C and the observed temperature is 10°C, then health agencies will have made the correct decision to not take preventative action, despite the large magnitude in forecast error. In this case, the MSE would be 100°C<sup>2</sup>. However, if we calculate a threshold weighted MSE (twMSE) that places an equal weight on decision thresholds above 35°C and a zero weight on decision thresholds below 35°C, the twMSE will give a value of 0 in this case. If at least one of the forecast value and the observed value is above 35°C and they differ from each other, then there will be a non-zero error.\n",
```

Comment:
```suggestion
    "As another example, suppose health agencies will start taking preventative action for treating heat-related illness whenever the temperature forecast for the following day exceeds 35°C and that this preventative action is increasingly important with even higher temperature forecasts. If the temperature forecast is 20°C and the observed temperature is 10°C, then health agencies will have made the correct decision to not take preventative action, despite the large magnitude in forecast error. In this case, the MSE would be 100°C<sup>2</sup>. However, if we calculate a threshold weighted MSE (twMSE) that places an equal weight on decision thresholds above 35°C and a zero weight on decision thresholds below 35°C, the twMSE will give a value of 0 in this case. If at least one of the forecast value and the observed value is above 35°C and they differ from each other, then there will be a non-zero error.\n",
```

Would it be worthwhile changing "heat illness" to "heat-related illness"? When I did a quick google, it seemed like heat-related illness may be more commonly used. But if "heat related" is the more appropriate term, no worries.

Reply:
Yes, I think that is a good suggestion - thanks @Steph-Chong